libgadget/forcetree.h

```diff
@@ -67,6 +66,8 @@ struct NODE
 typedef struct ForceTree {
     /*Is 1 if the tree is allocated. Only used inside force_tree_allocated() and when allocating.*/
     int tree_allocated_flag;
+    /* Flags that hmax has been computed for this tree*/
+    int hmax_computed_flag;
```

Comment:
This is only set but not used to branch operations?

Reply:
Yes - I put it in for debugging only. I have some future patches that use it to decide to do the hmax, but they are not useful yet.